src/main.rs

```diff
@@ -1,4 +1,147 @@
-fn main() {
-    println!("Hello, world!");
+use axum::{
+    async_trait,
+    extract::{FromRequest, Path, RequestParts},
+    handler::Handler,
+    http::StatusCode,
+    response::{IntoResponse, Json, Response},
+    routing::get,
+    Router, Server,
+};
+use serde_json::Value;
+use std::collections::HashMap;
+
+#[derive(Debug)]
+enum Version {
+    V1,
+}
+
+#[tokio::main]
+async fn main() {
+    let app = Router::new()
+        .route("/", get(root_handler))
+        .route("/ping", get(heartbeat_handler))
+        .route("/:version/contacts", get(contacts_handler))
+        .route("/:version/dictionary", get(dictionary_handler))
+        .route("/:version/faqs", get(faqs_handler))
+        .route("/:version/food/named/menu/pause", get(pause_menu_handler))
+        .route(
+            "/:version/printing/color-printers",
+            get(color_printers_handler),
+        )
+        .route("/:version/spaces/hours", get(hours_handler))
+        .route("/:version/tools/help", get(help_handler))
+        .route("/:version/transit/bus", get(transit_bus_handler))
+        .route("/:version/transit/modes", get(transit_modes_handler))
+        .route("/:version/webcams", get(webcams_handler))
+        .fallback(fallback.into_service());
+
+    Server::bind(&"0.0.0.0:3000".parse().unwrap())
+        .serve(app.into_make_service())
+        .await
+        .unwrap();
```

Comment:
Might be worth looking into removing these `unwrap`s, but this works well enough for now.